.ci.yaml

```diff
@@ -75,6 +75,99 @@ targets:
       target_file: mac_build_all_plugins.yaml
       channel: stable
 
+  ### iOS tasks ###
+  # TODO(stuartmorgan): Swap this and ios-build_all_plugins once simulator
+  # tests are reliable on the ARM infrastructure. See discussion at
+  # https://github.com/flutter/plugins/pull/5693#issuecomment-1126011089
+  - name: Max_x64 ios_platform_tests_0_4 master
```

Comment:
Without the clarify of "index" and "count" being in the name, I think it would be clear to 1-index them and add an "of". I.e., `ios_platform_tests_1_of_4` through `ios_platform_tests_4_of_4`.